src/main/java/pw/itr0/kaba/encrypt/PasswordBasedEncrypter.java

```diff
@@ -12,15 +17,32 @@
  * @since 1.0.0
  */
 public interface PasswordBasedEncrypter {
+
+    /**
+     * Returns {@link Cipher} to encrypt bytes with {@code password}.
+     *
+     * @param password encrypting password
+     * @return {@link Cipher} to be used on encrypt
+     * @throws InvalidKeyException if the {@code password} does not satisfy password specification of using encryption algorithm,
+     *                             or encryption algorithm using specified password length is not usable on current JDK
+     */
+    Cipher getEncryptionCipher(char[] password) throws InvalidKeyException;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/ryotan/kaba-vault/file/2671050500/issues/source?bid=3298589&fileBranchId=3332567#l29)